### ISSUE TITLE
Issues statistics

### ISSUE THREAD

**Issue by Amber1990Zhang** [vesoft-inc/github-statistics#34]
as title. Can I get the issues statistics?

**Comment by redbar0n** [comment]
yes, you will see it is implemented under https://vesoft-inc.github.io/github-statistics/#Issues

if this correspond to your request, please close this issue.
